### PR TITLE
[DE] Update `Community/osu!monthly`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
~~Regarding the CI, the link to `Community/osu!monthly#archives` needs to be adjusted in both English and German. Thus, it's probably out of scope here.~~

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Community/osu!monthly/de.md [hunk 1]
```diff
@@ -1,46 +1,37 @@
...
+| [The osu!monthly June 2011 Edition](https://osumonthly.com/pdf/osu!monthly-3.pdf) | 07.06.2011 |
+| [osu!monthly July 2011 Edition](https://osumonthly.com/pdf/osu!monthly-4.pdf) | 07.07.2011 |
+| [osu!monthly August 2011 Edition](https://osumonthly.com/pdf/osu!monthly-5.pdf) | 15.08.2011 |
+| [osu!monthly October 2011 Edition](https://osumonthly.com/pdf/osu!monthly-6.pdf) | 27.10.2011 |
 
-Weitere Informationen über die erste Version von osu!monthly sind auf der [Webseite von JJ](https://web.archive.org/web/20201112030400/http://jjwanda.com/projects/osumonthly.html) zu finden.
+## Neuauflage des Magazins (2017 bis 2018)
 
-### Season 2 (2017 bis 2018)
+Inspiriert von der ersten Version von osu!monthly, wurde dank ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) sowie ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) ein neues Team gebildet, mit dem Ziel, das Projekt fortzuführen. Von Oktober 2017 bis Februar 2018 wurden neue Editionen des Magazins veröffentlicht.
```
Comment (FreezyLemon) [inline wiki/Community/osu!monthly/de.md:22]: I think this wording can be a bit simpler without losing information:
```suggestion
Inspiriert von der ersten Auflage haben ::{ flag=GB }:: [mangomizer](https://osu.ppy.sh/users/1893718) sowie ::{ flag=US }:: [clayton](https://osu.ppy.sh/users/3666350) ein neues Team gebildet mit dem Ziel, das Projekt fortzuführen. Von Oktober 2017 bis Februar 2018 wurden neue Editionen des Magazins veröffentlicht.
```

File: wiki/Community/osu!monthly/de.md [hunk 2]
```diff
@@ -1,46 +1,37 @@
...
+
+Weitere Informationen sind auf [der offiziellen Webseite](https://osumonthly.com/) zu finden.
```
Comment (FreezyLemon) [inline wiki/Community/osu!monthly/de.md:24]: Probably not official in the typical sense:
```suggestion
Weitere Informationen sind auf [ihrer Webseite](https://osumonthly.com/) zu finden.
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
Only some small nitpicks

File: wiki/Community/osu!monthly/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Community/osu!monthly/de.md:22]: At first, I thought removing the second comma is a mistake. [Turns out, it is not](https://www.korrekturen.de/forum.pl/md/read/id/107873/sbj/komma-vor-mit-dem-ziel/).

Still sounds rough to me, but eh 🤷

File: wiki/Community/osu!monthly/de.md [hunk 3]
```diff
@@ -1,46 +1,37 @@
...
 
-Mit dem Wunsch, das Projekt zurückzubringen, wurde das [osu!monthly-Team](/wiki/People/osu!monthly_team) mit ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) an der Spitze gebildet, welches das Projekt in Form von Newsbeiträgen wieder zum Leben erweckt hat. Die Serie umfasste Ausgaben für den Zeitraum zwischen Juli 2022 und März 2023.
+## Ursprüngliche Auflage (2011)
 
-## Archiv
+Die ursprüngliche, von ::{ flag=US }:: [jjrocks](https://osu.ppy.sh/users/30985) und ::{ flag=GB }:: [Corin](https://osu.ppy.sh/users/639270) erstellte Auflage von osu!monthly lief von März bis Oktober 2011. Das Projekt begann als Google-Dokument, aber entwickelte sich schnell in ein PDF-Format, das einem gedruckten Magazin ähnelt. Diese Ausgabe war bekannt für ihr großes Spektrum an Themen sowie ihre unverwechselbare und künstlerische Gestaltung.
```
Comment (The-Last-Cookie) [inline wiki/Community/osu!monthly/de.md:7]:
```suggestion
Die ursprüngliche, von ::{ flag=US }:: [jjrocks](https://osu.ppy.sh/users/30985) und ::{ flag=GB }:: [Corin](https://osu.ppy.sh/users/639270) erstellte Auflage von osu!monthly lief von März bis Oktober 2011. Das Projekt begann als Google-Dokument, entwickelte sich aber schnell in ein PDF-Format, das einem gedruckten Magazin ähnelte. Diese Ausgabe war bekannt für ihr großes Spektrum an Themen sowie ihre unverwechselbare und künstlerische Gestaltung.
```
Liest sich etwas flüssiger.

File: wiki/Community/osu!monthly/de.md (hunk 2 above)
Comment (The-Last-Cookie) [reply wiki/Community/osu!monthly/de.md:24]: `ihrer` without any direct reference (only indirectly by the title `Neuauflage`) sounds weird to me in this context (thus I avoided it). How is this?

```suggestion
Weitere Informationen sind auf [der Webseite des Projekts](https://osumonthly.com/) zu finden.
```

Comment (FreezyLemon) [reply wiki/Community/osu!monthly/de.md:24]: good

**Review by FreezyLemon — APPROVED** [review]
LGTM

I think you can just change the broken wikilinks in this PR. But if you don't want to, just add `SKIP_WIKILINK_CHECK` to your PR description.

**Comment by The-Last-Cookie** [comment]
On `master`, the fragment was already removed, so I just edited the German file.

**Review by TicClick — APPROVED** [review]

**Pull request merged**